src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/ProcessHmcMessageService.java

```diff
@@ -67,6 +67,8 @@ public void processEventMessage(HmcMessage hmcMessage)
 
         SscsCaseData caseData = ccdCaseService.getCaseDetails(caseId).getData();
 
+        hearingUpdateService.resolveDwpState(hmcStatus, caseData);
```

Comment:
Shouldn't be returning something if you're not using it. Better to set it here instead of setting the dwp state then returning above.